assets/scripts/main.js

```diff
@@ -1,40 +1,41 @@
-let books = JSON.parse(localStorage.getItem('booksData')) || [];
+class Collection {
+  constructor() {
+    this.arr = JSON.parse(localStorage.getItem('booksData')) || [];
+  }
 
-function Book(title, author) {
-  this.title = title;
-  this.author = author;
-}
+  addBook(title, author) {
+    this.arr.push({ title, author });
+  }
```

Comment:
[OPTIONAL] The array methods like push, pop,  splice, etc can mutate/or change the array. Kindly consider using unmutabel methods like concat, filter, and slice.